pkg/operator/configobservation/interfaces.go

```diff
@@ -0,0 +1,27 @@
+package configobservation
+
+import (
+	"github.com/openshift/library-go/pkg/operator/resourcesynccontroller"
+	corelistersv1 "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
+
+	configlistersv1 "github.com/openshift/client-go/config/listers/config/v1"
+)
+
+type Listers struct {
+	ImageConfigLister  configlistersv1.ImageLister
+	BuildConfigLister  configlistersv1.BuildLister
+	ConfigMapLister    corelistersv1.ConfigMapLister
+	ImageConfigSynced  cache.InformerSynced
+	BuildConfigSynced  cache.InformerSynced
+	ConfigMapSynced    cache.InformerSynced
+	PreRunCachesSynced []cache.InformerSynced
+}
+
+func (l Listers) ResourceSyncer() resourcesynccontroller.ResourceSyncer {
+	return nil
```

Comment:
looks like openshift/cluster-openshift-apiserver-operator has non-nil returns, which drives the cache sync.
See apiserver [interfaces.go](https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/pkg/operator/configobservation/interfaces.go) and [observe_config_controller.go](https://github.com/openshift/cluster-openshift-apiserver-operator/blob/8aa4775bcff1c0e3d62606006cef70e4fda3d8f9/pkg/operator/configobservation/configobservercontroller/observe_config_controller.go#L46)

Reply:
this isn't about cache sync.  resourcesyncer copies resources between namespaces.  this operator does not use it anywhere in the code path.